### PR TITLE
Hide client ID and Secret using cocoapods-keys

### PR DESCRIPTION
**Pull request by Palleas**
Uses [cocoapods-keys](https://github.com/orta/cocoapods-keys) to obfuscate the client id and secret. The keys will be asked during next pod install. :boom:

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -16,4 +16,11 @@ pod 'JNWSpringAnimation'
...
-pod 'TTTAttributedLabel'
\ No newline at end of file
+pod 'TTTAttributedLabel'
+
+plugin 'cocoapods-keys', {
+  :project => "Exhibit",
+  :keys => [
+    "AubergisteClientID",
+    "AubergisteClientSecret"
+  ]}
```
Comment (remi) [inline Podfile]: Missing newline!

Comment (Palleas) [inline Podfile]: SORRY

**Comment by simondec** [comment]
Nice! :+1:

**Pull request merged**